packages/openapi-to-graphql/src/auth_builder.ts

```diff
@@ -37,7 +37,7 @@ type Viewer<TSource, TContext, TArgs> = {
   description: string
 }
 
-const translationLog = debug('translation')
```

Comment:
@swatkatz Okay, no problem! Just to let you know, you can produce the debug messages by setting your environment variables.

For example, if you append `DEBUG=translation` followed by whatever you want to do, e.g. `DEBUG=translation node myGraphQLServer.js`, you should be able to get all the `translation` messages.